### ISSUE TITLE
[APP] Dashboard

### ISSUE THREAD

**Issue by RodrigoRVSN** [RodrigoRVSN/set#4]
Include dashboard webviews

**Comment by RodrigoRVSN** [comment]
@mayoral-leonardo @gianlucabueno we can use [this webview library](https://github.com/react-native-webview/react-native-webview)

Import the webview

```js
import { WebView } from 'react-native-webview';
```

And use the webview, just setting the url of the page

```js
<WebView source={{ uri: 'URL_OF_DASHBOARD.COM '}} />
```

It'll render the dashboard with the respective URL provided by @nata-vito

**Comment by RodrigoRVSN** [comment]
Closed and done in #20
